README.rst

```diff
@@ -30,7 +30,7 @@ LiberTEM currently opens most file formats used for pixelated STEM:
 - Quantum Detectors MIB format (currently alpha, more testing and sample files highly appreciated)
 - Nanomegas .blo block files
 - Gatan K2IS raw format (currently beta)
-- FRMS6 from PNDetector pnCCD cameras (currently alpha, gain corection and windowing WIP)
+- FRMS6 from PNDetector pnCCD cameras (currently alpha, gain correction and windowing WIP)
```

Comment:
Windowing is working now, I guess?

Reply:
Yep, updated this to the current status in master.